packages/front-end/pages/api/init.ts

```diff
@@ -46,5 +47,6 @@ export default function handler(req: NextApiRequest, res: NextApiResponse) {
         ? "disable"
         : "enable",
     sentryDSN: NEXT_PUBLIC_SENTRY_DSN || "",
+    apiCredentials: ENABLE_API_CREDENTIALS || false,
```

Comment:
This is typed as a boolean, but here it's actually type `string | false` since environment variables are always strings.  Better to do:

```ts
apiCredentials: !!ENABLE_API_CREDENTIALS
```